<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineerDialog</class>
 <widget class="QDialog" name="EngineerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 125, 173);</string>
  </property>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>277</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>机柜注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>277</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>手动开柜</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SystemButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>277</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>277</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退    出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>EngineerDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
